src/ir_Corona.h

```diff
@@ -109,7 +110,6 @@ class IRCoronaAc {
   void setPower(const bool on);
   bool getPower();
   void setPowerButton(const bool on);
-  void updatePowerButton();
```

Comment:
This might be better as non public

Reply:
setPowerButton?

Reply:
updatePowerButton() is only used internally, so no need to be public

Reply:
And again, I'm not awake, sorry!

Reply:
Done